libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.h

```diff
@@ -66,6 +66,9 @@ NS_SWIFT_NAME(SalesforceLoginViewController)
  */
 @property (nonatomic, strong, nullable) IBOutlet UIView *oauthView;
 
+/** The biometric log in button */
+@property (nonatomic, strong, readonly, nullable) UIButton *biometricButton;
+
```

Comment:
Is this (going to be) used somewhere?

Reply:
Unknown, as of yet.  Stay tuned for a more final and review-worthy commit soon.